devel/verify-vim-wsdl.rb

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env ruby
+
+require 'active_support/core_ext/enumerable'
+require 'active_support/inflector'
+require "optimist"
+require "pathname"
+require "rbvmomi"
+require "wsdl/parser"
+
+def parse_args(args)
+  opts = Optimist.options do
+    banner <<~HELP
+      Usage:
+      verify-vim-wsdl.rb [path to wsdl] [path to vmodl.db]
+
+      --fix       Fix the wsdl types
+      --help, -h  Print this message and exit
+    HELP
```

Comment:
Instead of using banner you can just define the usage line, and the rest is automatic.

Reply:
:+1: done